### PR TITLE
tap.rb: remove defunc dupes

### PR DESCRIPTION
**Pull request by maxim-belkin**
Removing dupes from tap.rb

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Travis on macOS:
```
HEAD is now at 9c5116a cleanup_shared: Do not clean vendor/portable-ruby [Linux] (#39)
Error: No such file or directory @ rb_sysopen - /Users/travis/build/Linuxbrew/brew/Library/Homebrew/test/support/fixtures/testbottest.rb
```

**Comment by sjackman** [comment]
The failure on TravisCI on macOS has been around for a long time. Fixing it requires merging homebrew-test-bot from Homebrew into Linuxbrew.

File: Library/Homebrew/tap.rb [hunk 1]
```diff
@@ -127,12 +127,7 @@ def default_remote
     if OS.mac?
       "https://github.com/#{full_name}"
     else
-      case "#{user}/#{repo}"
-      when "Homebrew/dupes"
-        "https://github.com/Linuxbrew/homebrew-#{repo}"
-      else
-        "https://github.com/#{user}/homebrew-#{repo}"
-      end
+      "https://github.com/#{user}/homebrew-#{repo}"
```
Comment (sjackman) [inline Library/Homebrew/tap.rb]: Change this to:
```
  def default_remote
    "https://github.com/#{full_name}"
  end
```

**Review by sjackman — CHANGES_REQUESTED** [review]
Thanks, Maxim!

**Comment by iMichka** [comment]
Please wait until #514 is merged before merging this :)

File: Library/Homebrew/tap.rb (hunk 1 above)
Comment (sjackman) [reply Library/Homebrew/tap.rb]: @maxim-belkin Could you please make this change when you get to it?

Comment (maxim-belkin) [reply Library/Homebrew/tap.rb]: done!

**Comment by sjackman** [comment]
Merged. Thanks, Maxim!

**Pull request merged**